src/registry/metadataRegistry.json

```diff
@@ -4687,6 +4688,14 @@
       "directoryName": "PublicKeyCertificateSet",
       "inFolder": false,
       "strictDirectoryName": false
+    },
+    "transactionprocessingtype": {
+      "id": "transactionprocessingtype",
+      "name": "TransactionProcessingType",
+      "suffix": "transactionProcessingTypes",
+      "directoryName": "transactionProcessingType",
```

Comment:
This doesn't match your server side definition.  The directoryName is "transactionProcessingTypes".

Why is the suffix plural?  It should really be singular.